src/JuMP.jl

```diff
@@ -702,15 +712,15 @@ a warning is generated once.
 function operator_warn(::AbstractModel) end
 function operator_warn(model::Model)
     model.operator_counter += 1
-    if model.operator_counter > 20000
-        Base.warn_once(
+    if model.operator_counter > OPERATOR_WARN_LIMIT
```

Comment:
This introduces a type inference issue. Maybe `OPERATOR_WARN_LIMIT::Int` here and `function set_operator_warn_limit(new_limit::Int)` above?

Reply:
Is it because OPERATOR_WARN_LIMIT is a non-const module level global?
It only works on JuMP.Model-s anyway - could make a Model field?

Reply:
Would have the benefit of making clear the global-ish behaviour that this has already.

Reply:
In fact, later in this file (test/operators.jl), something similar to that happens!
I'm going to integrate the princple of this change into that, and drop the global.